### PR TITLE
Add radioactivedecay

### PR DESCRIPTION
**Pull request by alexmalins**
Add conda-forge recipe for radioactivedecay PyPI package

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

I am happy to be listed as a maintainer for this recipe

### CODE REVIEW

**Comment by alexmalins** [comment]
Looks like I need to add a MANIFEST.in to the project repository so the LICENSE file gets properly packaged, and fix some linting issues. Please bear with me while I fix everything.

File: recipes/radioactivedecay/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/radioactivedecay-{{ version }}.tar.gz
+  sha256: ddb0a1054cf659db465a7355be3f8ac203e81206aa90b07798ac0eeafb65d99a
+
+build:
+  number: 0
+  skip: true   # [py<36]
```
Comment (dopplershift) [inline recipes/radioactivedecay/meta.yaml:15]: This looks like a pure Python package. You can just change this to:
```suggestion
  noarch: python
```

File: recipes/radioactivedecay/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - importlib_resources  # [py<37]
```
Comment (dopplershift) [inline recipes/radioactivedecay/meta.yaml:23]: In this case it's best to just always depend on `importlib_resources`.  Or you can remove it entirely and just make the dependencies on Python as `>=3.7`. You'll need to at least make them `>=3.6` when making the recipe noarch.

File: recipes/radioactivedecay/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    - python -m unittest discover
+  requires:
+    - pip
+    - python
+  source_files:
+    - tests/
+
+about:
+  home: https://github.com/alexmalins/radioactivedecay
+  license: MIT
```
Comment (dopplershift) [inline recipes/radioactivedecay/meta.yaml:45]: Probably should also note the data file license:
```suggestion
  license:
    - MIT
    - LicenseRef-ICRP-07
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
This looks in generally good shape, just a few items to take care of.

File: recipes/radioactivedecay/meta.yaml (hunk 1 above)
Comment (alexmalins) [reply recipes/radioactivedecay/meta.yaml]: Done thanks

File: recipes/radioactivedecay/meta.yaml (hunk 2 above)
Comment (alexmalins) [reply recipes/radioactivedecay/meta.yaml]: Thanks, opted for the second option so the allowed Python versions (>=3.6) are the same on PyPI and on conda-forge.

File: recipes/radioactivedecay/meta.yaml (hunk 3 above)
Comment (alexmalins) [reply recipes/radioactivedecay/meta.yaml:45]: Done thanks. conda-build did not like the list formatting, but worked with multiple `license:` and `license_family:` entries.

Thanks for the review and the suggestions, @dopplershift !

File: recipes/radioactivedecay/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+test:
+  imports:
+    - radioactivedecay
+  commands:
+    - pip check
+    - python -m unittest discover
+  requires:
+    - pip
+    - python >=3.6
```
Comment (chrisburr) [inline recipes/radioactivedecay/meta.yaml:39]: No need for this
```suggestion
```

Comment (alexmalins) [reply recipes/radioactivedecay/meta.yaml]: thanks @chrisburr

File: recipes/radioactivedecay/meta.yaml (hunk 3 above)
Comment (chrisburr) [reply recipes/radioactivedecay/meta.yaml:44]: You need to use an "spdx license expression" (basically `XXX OR YYY` or `XXX AND YYY`, I guess `AND` applies here?)

Comment (chrisburr) [reply recipes/radioactivedecay/meta.yaml:44]: You can then drop `license_family` now as it's automatically generated for valid SPDX expressions in `license`

**Review by chrisburr — CHANGES_REQUESTED** [review]
Just so it doesn't get missed, the license metadata still needs fixing: https://github.com/conda-forge/staged-recipes/pull/14902#discussion_r632822813

File: recipes/radioactivedecay/meta.yaml (hunk 3 above)
Comment (alexmalins) [reply recipes/radioactivedecay/meta.yaml]: Fixed, thanks @chrisburr

**Pull request merged**